unit-tests/3D/test-projection-from-recording.py

```diff
@@ -2,7 +2,7 @@
 # Copyright(c) 2023 Intel Corporation. All Rights Reserved.
 
 #temporary fix to prevent the test from running on Win_SH_Py_DDS_CI
-#test:donotrun:dds
+#test:donotrun
```

Comment:
Can you try to keep both and on the new one add a comment that we disable it until LRS-XXX ticket is resolved due to stability issues?
This will help us remove 1 line to reactivate it in the future and not forget about the DDS issue we wanted to avoid

Reply:
done